benchmark/src/main/java/ai/timefold/solver/benchmark/impl/result/SingleBenchmarkResult.java

```diff
@@ -49,6 +49,7 @@ public class SingleBenchmarkResult implements BenchmarkResult {
     private Integer failureCount = null;
     private Score totalScore = null;
     private Score averageScore = null;
+    private boolean hasUninitializedScores = false;
```

Comment:
In some places, we use a flag that returns true when the score is initiated. What do you think  abount maintaining this pattern and introducing a flag called `allScoresInitialized`?